components/partials/modals/bridge/index.vue

```diff
@@ -221,6 +226,20 @@ export default Vue.extend({
       return this.$t('bridge.selectDestinationNetwork')
     },
 
+    hasAllowance(): boolean {
+      const { erc20TokensWithBalanceAndPriceFromBank, form } = this
+
+      const token = erc20TokensWithBalanceAndPriceFromBank.find(
+        ({ denom }) => denom === form.token.denom
+      )
+
+      if (!token) {
+        return false
+      }
+
+      return token.allowance !== '0'
```

Comment:
Ah, let's not compare things like this, let's use `new BigNumberInBase(token.allowance).gt(0)`